<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>181</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="leMoney"/>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pb500">
        <property name="text">
         <string>500</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb100">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb50">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb10">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pbCoffee">
        <property name="text">
         <string>Coffee200</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbTea">
        <property name="text">
         <string>Tea100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbYul">
        <property name="text">
         <string>Yul250</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pbReset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>pb500</zorder>
  <zorder>pb100</zorder>
  <zorder>pb50</zorder>
  <zorder>pb10</zorder>
  <zorder>pbCoffee</zorder>
  <zorder>pbTea</zorder>
  <zorder>pbYul</zorder>
  <zorder>pbReset</zorder>
  <zorder>pb500</zorder>
  <zorder>leMoney</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
